app/api/task_sheet/task_sheet.tsx

```diff
@@ -0,0 +1,8 @@
+export interface TaskSheet {
+    id: number;
```

Comment:
// Add a line to include tasks in the TaskSheet interface
  // tasks: Task[];

Reply:
@veryyynice @johan253